Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -87,9 +87,9 @@ public void Actions_WhenShortcutsDisabled_AllConflictingActionsTrigger()
         Press(keyboard.wKey);
 
         // All Actions were triggered
-        Assert.That(action1.WasPerformedThisFrame());
```

Comment:
There are a lot of tests update to make this pass?
Were all these failing without the changes?

Make me worry about how much customer code would be impacted by the change.